conda/core/subdir_data.py

```diff
@@ -208,14 +235,16 @@ def url_w_repodata_fn(self):
 
     @property
     def cache_path_json(self):
-        return self.cache_path_base + ("1" if context.use_only_tar_bz2 else "") + ".json"
+        return Path(self.cache_path_base + ("1" if context.use_only_tar_bz2 else "") + ".json")
 
     @property
     def cache_path_state(self):
         """
         Out-of-band etag and other state needed by the RepoInterface.
         """
-        return self.cache_path_base + ".state.json"
+        return Path(
+            self.cache_path_base + ("1" if context.use_only_tar_bz2 else "") + ".state.json"
+        )
```

Comment:
I agree that everything is state so that's not very helpful, OTOH mamba and conda 23.1 are currently using this filename. Feel free to submit a PR to https://github.com/conda-incubator/ceps/pull/46

Reply:
Done! https://github.com/conda-incubator/ceps/pull/46#discussion_r1123459101

Reply:
There seems to have been consensus for `path.json`? https://github.com/conda-incubator/ceps/pull/46#discussion_r1127046616